ERRATA.md

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
+
+## Errata
+
+### 2020-11-30: Document inconsistency
+This bug existed between 2020-08-28 and 2020-09-23.
```

Comment:
We should probably use versions here. Introduced in Vespa 7.277.38. Fixed in 7.292.82.

Reply:
better!